src/Agent.Plugins/PipelineArtifact/PipelineArtifactServer.cs

```diff
@@ -171,22 +171,40 @@ private BuildDropManager CreateBulidDropManager(AgentTaskPluginExecutionContext
             string[] minimatchFilters,
```

Comment:
>string[] [](start = 12, length = 8)

Should be IEnumerable,

because string[] contact implies you can modify the array elements

Reply:
Doesn't apply anymore